src/api/create-vis.js

```diff
@@ -75,6 +76,13 @@ var loadVizJSON = function (el, visModel, vizjsonData, options) {
     layerSelectorEnabled = options.layerSelectorEnabled;
   }
 
+  var renderMode = RenderModes.AUTO;
```

Comment:
cc: @donflopez @javisantana I kept `vector` as the option for CartoDB.js for now. But I introduced the concept of render modes internally.